src/components/DragAndDrop/Provider/types.ts

```diff
@@ -8,7 +8,7 @@ type DragAndDropProviderProps = {
     isDisabled?: boolean;
 
     /** Indicate that users are dragging file or not */
-    setIsDraggingOver: (value: boolean) => void;
+    setIsDraggingOver?: (value: boolean) => void;
```

Comment:
Are you sure this should be optional?

Reply:
@blazejkustra Yep, it's already so [on the main](https://github.com/Expensify/App/blob/main/src/components/DragAndDrop/Provider/types.ts#L11). Report [screen](https://github.com/Expensify/App/blob/main/src/pages/home/ReportScreen.tsx#L516) uses `DragAndDropProvider` without `setIsDraggingOver` as well.